src/helpers/personas/fixSituationWithPartialMosaic.ts

```diff
@@ -6,7 +6,7 @@ import { PublicodesExpression } from 'publicodes'
 type Props = {
   situation: Situation
   everyMosaicChildrenWithParent: Record<DottedName, DottedName[]>
-  safeGetRule: (rule: DottedName) => NGCRuleNode | null
+  safeGetRule: (rule: DottedName) => NGCRuleNode | undefined
```

Comment:
Just curious. What was wrong with `null` ? It generates some useless diff and ternaries below...

Reply:
I hate `null`, pleading for the use of `undefined` everywhere it's needed to provide an empty value, as it's matching the default javascript behaviour. Undefined also seems to work better with typescript ; here I had a type issue to fix where I used typescript optional parameter.

Reply:
Yes but by hating `null`, you add more if and else to hide the engine returning `null`. The thing is. We should fix engine signature.

(I also hate null...)